torch/testing/_internal/opinfo/core.py

```diff
@@ -38,6 +41,9 @@
 from torchgen.utils import dataclass_repr
 
 
+# setup logging
+log = logging.getLogger(__name__)
```

Comment:
Oh do we have logging as our normal print infra for tests now?

Reply:
ah I'm probably doing this wrong then. What's the right way to do a optional debug-only log entry?

Reply:
This i have no idea....maybe this is the way? cc @zou3519